src/USER-QUIP/pair_quip.h

```diff
@@ -24,7 +24,7 @@ PairStyle(quip,PairQUIP)
 
 extern "C"
 {
-   void quip_lammps_wrapper(int*, int*, int*,
+   void quip_lammps_wrapper(int*, int*, int*, int*,
```

Comment:
please note, that this change will break when compiling LAMMPS with -DLAMMPS_BIGBIG where `tagint` will be a `long int` instead of an `int`, i.e. 64-bit instead of 32-bit.

So you either need to change your interface or disallow including QUIP in LAMMPS with -DLAMMPS_BIGBIG.